nologin: extend the /run/nologin descriptions a bit

poettering opened the pull request:
This is an attempt to improve #8228 a bit, by extending the /run/nologin
a bit, but still keeping it somewhat brief.

On purpose I used the vague wording "unprivileged user" rather than
"non-root user" so that pam_nologin can be updated to disable its
behaviour for members of the "wheel" group one day, and our messages
would still make sense.

Fixes: #8228

keszybz commented:
Yeah, I guess this can go in, irrespectively of any further changes to the /run/nologin logic. But please rebase.

poettering commented:
I force pushed a new, rebased version. It is slightly extended: I implemented a common function now for writing the file, so that the message text is only listed once in our codebase, so that it is less likely to get out of sync.

keszybz commented:
I replaced "Fixes" with "See", because I think #8228 needs a bit more discussion. We can close it later.
